extra/linux/org.alacritty.Alacritty.appdata.xml

```diff
@@ -1,34 +1,45 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Copyright 2016-2022 Joe Wilm, The Alacritty Project Contributors -->
+<!-- SPDX-License-Identifier: MIT -->
 <component type="desktop-application">
   <id>org.alacritty.Alacritty</id>
-  <!-- Translators: The application name -->
+
+  <developer_name>Christian Duerr</developer_name>
```

Comment:
```suggestion
  <developer_name>Christian Duerr</developer_name>
  <developer id="org.alacritty">
    <name>Christian Duerr</name>
  </developer>
```

Let's also add a modern "developer" tag. This fixes the remaining warnings by AppStream validators and makes the metadata file future-proof. :-)

The old "developer_name" tag is kept for compatibility with older parsers (it is safely ignored by newer ones).